widget.php

```diff
@@ -512,17 +514,22 @@ protected function form_template_groups($instance, $groups) {
           <p>
             <input list="emfluence-emailer-groups-list"/>
             <button type="button" onclick="emfluenceEmailerWidget.groups.add(this)">' . __('Add') . '</button>
+            <!-- Subrata -->
+            <input class="begroups" type="text" id="' . $this->get_field_id( 'begroups' ) . '" name="' . $this->get_field_name( 'begroups' ) . '" value="' . $instance['begroups'] . '" style="display:none;" />
```

Comment:
Should we escape `$instance['begroups']` for output?

Reply:
Done.